screen.py

```diff
@@ -18,6 +22,7 @@
 HEIGHT_OF_SCOREBOARD = 200
 SCORE_BOARD = pygame.Surface((screen.get_width(), HEIGHT_OF_SCOREBOARD))
 FONT = pygame.font.SysFont('comicsansms', 30)
+largeFONT = pygame.font.Font('freesansbold.ttf', 40)
```

Comment:
What about global vars thats can change... like move_turn in main?

Reply:
Usually you should try to avoid global variables, because you can change them from anywhere in your project and it can lead to painful bugs.
But, google's convention to global variables names are global_name, just as you did 👍 
https://google.github.io/styleguide/pyguide.html This is a long document on google's conventions for writing python code, you could read some parts of it if you want.

Reply:
:100: